insights/specs/insights_archive.py

```diff
@@ -14,7 +14,6 @@ class InsightsArchiveSpecs(Specs):
     auditctl_status = simple_file("insights_commands/auditctl_-s")
     aws_instance_id_doc = simple_file("insights_commands/python_-m_insights.tools.cat_--no-header_aws_instance_id_doc")
     aws_instance_id_pkcs7 = simple_file("insights_commands/python_-m_insights.tools.cat_--no-header_aws_instance_id_pkcs7")
-    aws_instance_type = simple_file("insights_commands/python_-m_insights.tools.cat_--no-header_aws_instance_type")
```

Comment:
Hey @xiangce  this needs to stay since we have existing archives that include this spec.

Reply:
Done, @bfahr , Thanks.